UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs

```diff
@@ -186,5 +186,20 @@ public IActionResult GetBatchStatus([FromRoute, Required] string batchId)
             }
             return Unauthorized();
         }
+
+        [HttpPost]
+        [Route("/cleanUp")]
+        public IActionResult CleanUp([FromBody] List<string> batchId)
+        {
+            if (batchId != null)
```

Comment:
Add count check to if condition
if (batchId != null && batchId.Count > 0)

Reply:
Thanks for highlighting it, making the changes accordingly.